content/docs/0.8.1/snapshots-and-backups/backup-and-restore/set-backup-target.md

```diff
@@ -112,6 +113,11 @@ We provides two testing purpose backupstore based on NFS server and Minio S3 ser
 
 **Result:** Longhorn can store backups in S3. To create a backup, see [this section.](../create-a-backup)
 
+### Using a self-signed SSL certificate for S3 communication
+If you want to use a self-signed ssl certificate you can expose an `AWS_CERT` environment variable that contains your public certificate to the longhorn pods, or see the example in [Set up a Local Testing Backupstore](#set-up-a-local-testing-backupstore).
```

Comment:
`SSL` not `ssl`.

I don't feel we want to mention `environment variable` here. `AWS_CERT` is a part of the secret, not an environment variable. Using the secret's content as environment variables is how we implemented it internally, so the user doesn't need to know. Otherwise, they might be confused.

Reply:
How about I remove all that and just go with see the example in...
As well as the note about the type, etc?

Reply:
How about
> If you want to use a self-signed SSL certificate, you can specify `AWS_CERT` in the Kubernetes secret that you provided to Longhorn. See the example in [Set up a Local Testing Backupstore](#set-up-a-local-testing-backupstore).